tests/modules/nf-core/fastqscreen/buildfromindex/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: "fastqscreen buildfromindex"
+  command: nextflow run ./tests/modules/nf-core/fastqscreen/buildfromindex -entry test_fastqscreen_buildfromindex -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/fastqscreen/buildfromindex/nextflow.config
+  tags:
+    - "fastqscreen"
+    - "fastqscreen/buildfromindex"
+  files:
+    - path: "output/fastqscreen/FastQ_Screen_Genomes"
+    # md5sum: e667c7caad0bc4b7ac383fd023c654fc
```

Comment:
Is this commented out? Sorry, reviewing on my phone so not entirely sure.

Reply:
It is removed now.